diff --git a/scripts/irc b/scripts/irc
index 3557d7a..ad16dc1 100755
--- a/scripts/irc
+++ b/scripts/irc
@@ -1,5 +1,5 @@
 #! /bin/sh
-## Shit-simple script to supply the "privmsg <recipient>" of IRC typein, and
+## A simple script to supply the "privmsg <recipient>" of IRC typein, and
 ## keep the connection alive.  Pipe this thru "nc -v -w 5 irc-server port".
 ## Note that this mechanism makes the script easy to debug without being live,
 ## since it just echoes everything bound for the server.
diff --git a/scripts/probe b/scripts/probe
index c47dc3f..eaa1ad5 100755
--- a/scripts/probe
+++ b/scripts/probe
@@ -1,5 +1,5 @@
 #! /bin/sh
-## launch a whole buncha shit at yon victim in no particular order; capture
+## launch various probes at yon victim in no particular order; capture
 ## stderr+stdout in one place.  Run as root for rservice and low -p to work.
 ## Fairly thorough example of using netcat to collect a lot of host info.
 ## Will set off every intrusion alarm in existence on a paranoid machine!
@@ -42,7 +42,7 @@ echo 'END big udp'
 $UCMD -r -z "$1" 175-180 2000-2003 530-533 1524 1525 666 213 8000 6250 2>&1
 # Use our identd-sniffer!
 iscan "$1" 21 25 79 80 111 53 6667 6000 2049 119 2>&1
-# this gets pretty intrusive, but what the fuck.  Probe for portmap first
+# this gets pretty intrusive.  Probe for portmap first
 if nc -w 5 -z -u "$1" 111 ; then
   showmount -e "$1" 2>&1
   rpcinfo -p "$1" 2>&1
diff --git a/scripts/web b/scripts/web
index 382b18e..36f0eec 100755
--- a/scripts/web
+++ b/scripts/web
@@ -3,10 +3,10 @@
 ## tiny dismal kludges all band-aided together, and now these bottom-line
 ## clueless pinheads who never heard of "TCP handshake" want to run
 ## *commerce* over the damn thing.  Ye godz.  Welcome to TV of the next
-## century -- six million channels of worthless shit to choose from, and
+## century -- six million channels of worthless junk to choose from, and
 ## about as much security as today's cable industry!
 ##
-## Having grown mightily tired of pain in the ass browsers, I decided
+## Having grown mightily tired of useless browsers, I decided
 ## to build the minimalist client.  It doesn't handle POST, just GETs, but
 ## the majority of cgi forms handlers apparently ignore the method anyway.
 ## A distinct advantage is that it *doesn't* pass on any other information
diff --git a/scripts/webproxy b/scripts/webproxy
index cee2d29..1c756fb 100755
--- a/scripts/webproxy
+++ b/scripts/webproxy
@@ -89,7 +89,7 @@ if test "$1" ; then
   echo "=== Launch: $CLIENT" >> $LFILE
   echo "Proxy running -- will accept connections on $PORT from $CLIENT"
   echo "  Logging queries to $LFILE"
-  test -f "$CFILE" && echo "  and connection fuckups to $CFILE"
+  test -f "$CFILE" && echo "  and connection errors to $CFILE"
 
 # and run the first listener, showing us output just for the first hit
   nc -v -w 600 -n -l -p $PORT -e "$0" $XNC "$CLIENT" &
@@ -125,7 +125,6 @@ if test "$x3" ; then
 fi
 # check for non-GET *after* we log the query...
 test "$x1" != "GET" && echo "sorry, this proxy only does GETs" && exit 0
-# no, you can *not* phone home, you miserable piece of shit
 test "`echo $hh | fgrep -i netscap`" && \
   echo "access to Netscam's servers <b>DENIED.</b>" && exit 0
 # Do it.  30 sec net-wait time oughta be *plenty*...
diff --git a/scripts/websearch b/scripts/websearch
index f721625..73c6b67 100755
--- a/scripts/websearch
+++ b/scripts/websearch
@@ -31,7 +31,7 @@ echo '' ; echo "=== Infoseek ==="
 echo "  is broken."
 # doquery "WW/IS/Titles?qt=${PLUSARG}" www2.infoseek.com 80
 # ... which doesn't work cuz their lame server wants the extra newlines, WITH
-# CRLF pairs ferkrissake.  Fuck 'em for now, they're hopelessly broken.  If
+# CRLF pairs ferkrissake.  They're hopelessly broken.  If
 # you want to play, the basic idea and query formats follow.
 # echo "GET /WW/IS/Titles?qt=${PLUSARG}" > $IFILE
 # echo "" >> $IFILE
@@ -48,12 +48,12 @@ echo '' ; echo "=== Inktomi ==="
 doquery "/query/?query=${PLUSARG}&hits=100" ink3.cs.berkeley.edu 1234 | \
   sed '/^<IMG ALT.*inktomi.*\.gif">$/d'
 
-#djnews lame shit limits hits to 120 and has nonstandard format
+#djnews limits hits to 120 and has nonstandard format
 echo '' ; echo "=== Dejanews ==="
 doquery "/cgi-bin/nph-dnquery?query=${PIPEARG}+maxhits=110+format=terse+defaultOp=AND" \
   smithers.dejanews.com 80
 
-# OLD lycos: used to work until they fucking BROKE it...
+# OLD lycos: used to work until they BROKE it...
 # doquery "/cgi-bin/pursuit?query=${PLUSARG}&maxhits=300&terse=1" \
 #   query5.lycos.cs.cmu.edu 80
 # NEW lycos: wants the User-agent field present in query or it returns nothing
@@ -62,7 +62,7 @@ doquery "/cgi-bin/nph-dnquery?query=${PIPEARG}+maxhits=110+format=terse+defaultO
 echo \
 "GET /cgi-bin/pursuit?query=${PLUSARG}&maxhits=300&terse=terse&matchmode=and&minscore=.5 HTTP/1.x" \
   > $IFILE
-echo "User-agent: *FUCK OFF*" >> $IFILE
+echo "User-agent: dummy" >> $IFILE
 echo "Why: go ask todd@pointcom.com (Todd Whitney)" >> $IFILE
 echo '' >> $IFILE
 echo '' ; echo "=== Lycos ==="
